### PR TITLE
Add Alertmanager Guide

### PR DESCRIPTION
**Pull request by benjaminhuo**
Signed-off-by: Benjamin <benjamin@yunify.com>

### CODE REVIEW

File: content/en/monitoring/alertmanager.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+In KubeSphere v3.0 and above, user can also use Alertmanager to manage alerts triggered by K8s/KubeSphere audit events. For more details, please refer to [KubeSphere Auditing](../audit/kube-auditing.md)
+
+## Receiving notifications for Alertmanager alerts
+
+Usually to receive notifications for Alertmanager alerts, user has to edit Alertmanager's configuration file manually to configure receiver settings like Email/Slack etc.
+
+This is not convenient for K8s users and breaks the multi-tenancy principle/architecture of KubeSphere because alerts triggered by workloads in different namespaces belong to different users could be sent to the same user's email address.
+
+To use Alertmanager to manage alerts in KubeSphere while complying with the multi-tenancy principle and providing K8s user friendly notification experience, KubeSphere open sourced [Notification Manager](https://github.com/kubesphere/notification-manager), a K8s native notification management tool with multi-tenancy support. User can select to install this tool in KubeSphere v3.0 and above.
```
Comment (FeynmanZhou) [inline content/en/monitoring/alertmanager.md:31]: KubeSphere open sourced [Notification Manager](https://github.com/kubesphere/notification-manager) => KubeSphere developed [Notification Manager](https://github.com/kubesphere/notification-manager), a K8s native notification management tool with multi-tenancy support; and make it 100% open source.

Comment (Sherlock113) [reply content/en/monitoring/alertmanager.md:31]: My understanding and suggestion for this paragraph is:

This is not convenient for K8s users as it breaks the multi-tenant principle/architecture of KubeSphere. More specifically, alerts triggered by workloads in different namespaces should have been sent to different users accordingly, while they might be sent to the same user.

File: content/en/monitoring/alertmanager.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Use Alertmanager to manage alerts sent from KubeSphere auditing 
+
+In KubeSphere v3.0 and above, user can also use Alertmanager to manage alerts triggered by K8s/KubeSphere audit events. For more details, please refer to [KubeSphere Auditing](../audit/kube-auditing.md)
+
+## Receiving notifications for Alertmanager alerts
+
+Usually to receive notifications for Alertmanager alerts, user has to edit Alertmanager's configuration file manually to configure receiver settings like Email/Slack etc.
+
+This is not convenient for K8s users and breaks the multi-tenancy principle/architecture of KubeSphere because alerts triggered by workloads in different namespaces belong to different users could be sent to the same user's email address.
```
Comment (FeynmanZhou) [inline content/en/monitoring/alertmanager.md:29]: This sentence has grammar issues, please refer to @Sherlock113 's comment.

Comment (Sherlock113) [reply content/en/monitoring/alertmanager.md:29]: My understanding and suggestion for this paragraph is:

This is not convenient for K8s users as it breaks the multi-tenant principle/architecture of KubeSphere. More specifically, alerts triggered by workloads in different namespaces should have been sent to different users accordingly, while they might be sent to the same user.

Comment (benjaminhuo) [reply content/en/monitoring/alertmanager.md:29]: not convenient for K8s users and break the multi-tenacy principle are two separated points and has no connections actually. I'll see how to make it short

**Comment by benjaminhuo** [comment]
@FeynmanZhou @Sherlock113 , I've made some changes to the guide

**Pull request merged**